eng/common/scripts/Verify-AgentOS.ps1

```diff
@@ -8,7 +8,7 @@ function Throw-InvalidOperatingSystem {
     throw "Invalid operating system detected. Operating system was: $([System.Runtime.InteropServices.RuntimeInformation]::OSDescription), expected image was: $AgentImage"
 }
 
-if ($IsWindows -and $AgentImage -match "windows|win|MMS2019") {
+if ($IsWindows -and $AgentImage -match "windows|win|MMS\d{4}") {
```

Comment:
@mikeharder not sure we care to do it with this change but I wonder if we should call the image names the same as the public names and skip on MMS prefix for our usages. Especially given that we don't actually end up using that name when from our pipeline files as it is mostly just documentation.